testsuite/features/secondary/min_project_lotus.feature

```diff
@@ -15,55 +15,57 @@ Feature: Project Lotus
   Scenario: Pre-requisite: Create custom channel for PTFs
     When I follow the left menu "Software > Manage > Channels"
     And I follow "Create Channel"
-    And I enter "Custom Channel for SLES15SP4 PTFs" as "Channel Name"
-    And I enter "sles15sp4-ptfs" as "Channel Label"
+    And I enter "Custom Channel for SLES15SP7 PTFs" as "Channel Name"
+    And I enter "sles15sp7-ptfs" as "Channel Label"
     And I select the parent channel for the "sle_minion" from "Parent Channel"
     And I enter "Custom channel for PTFs" as "Channel Summary"
     And I uncheck "gpg_check"
     And I click on "Create Channel"
-    Then I should see a "Custom Channel for SLES15SP4 PTFs" text
+    Then I should see a "Custom Channel for SLES15SP7 PTFs" text
 
   Scenario: Pre-requisite: Create custom repository for PTFs
     When I follow the left menu "Software > Manage > Repositories"
     And I follow "Create Repository"
-    And I enter "sles15sp4_ptf_repo" as "label"
+    And I enter "sles15sp7_ptf_repo" as "label"
     And I enter "http://updates.suse.de/download/PTF/Release/A127499/SLES/15.4/x86_64/ptf/" as "url"
```

Comment:
What about adding a TODO comment here, to remind us that we need to change the repo?

Reply:
The todo is added